src/schema/README.md

```diff
@@ -399,6 +399,13 @@ These objects additionally have the field:
 | `type`   | The type (such as `string`, `integer`, `object`) of values the term describes                                                                                                                                   |
 | `format` | The format of the term (defined in `objects.formats`)                                                                                                                                                           |
 
+`objects.columns` additionally permit a `definition` field that may take any value permissible
+for the definition of a column in a JSON sidecar:
+
+| Field        | Description                                                                                                                                                                                                   |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `definition` | A JSON object that describes the column as in [Common principles - Tabular files][tabular files]. This is mutually exclusive with the `type` field, and is used for fields that have overridable definitions. |
```

Comment:
not clear to me what "overridable definitions" -- overridable by a BIDS user to override Units etc?  so not all columns can have Units overridden?

Reply:
Does reading #1838 clarify? I tried to be clear here, and I'm happy to improve the wording, but if we're not starting from the same baseline understanding, we're in for many rounds of back and forth.

Reply:
Thanks! It does provide more background indeed. I formulated separate
- https://github.com/bids-standard/bids-specification/issues/2048
for potential further "clarification" which should not be  a part of this PR which just to improve documentation. I think your proposed formulation is as good as it gets until we address #2048 one way or another and then either adjust schema or description here to use consistent terminology (e.g. "fully-defined", "conventional" or alike).

Reply:
btw -- how does it relate to

```
`*`            | JSON-schema fields to further constrain values                      |
```
?  is JSON-schema here the https://json-schema.org/ or the "JSON sidecar schema" or ... ?

Reply:
Yes, we use actual JSON schema, and the JS validator uses the AJV package to perform validation. We have tried to adopt JSON schema as much as is practical.